### ISSUE TITLE
How to get Projects hidden from Budget Overview? / Usage Problem? 

### ISSUE THREAD

**Issue by Skeeve** [kimai/kimai#705]
Hi!

I like Kimai very much but I think I don't use it correctly ;) I found the forums to be kind of "defunct" as no registration seems to be possible anymore. I also asked this in kimais/documentation https://github.com/kimai/documentation/issues/5 but now realized thet the forums explictily link to here. Please tell me if I should ask this question elsewhere.

In my current job, I get a certain amount of days assigned for different projects. As there is no money (directly) involved, I simply set my rate to 60 and set the budget to

    60 (minutes) * 8 (hours per day) * No. of Days 

This works fine and gives me an overview in the budget tab about the amount of time used per project.

But now we are at the end of the month and I need to "reset" everything for the next month.

My current approach is to append "YYYY-MM " (so "2016-05 " this month) to the project name. The downside for me is: I cannot hide, in the budget view, all the old "2016-XX " projects.

Do you have any suggestion how I could approach this?

**Comment by kevinpapst** [comment]
Hey @Skeeve ,
you could check out the upcoming version, where I worked on the Budget tab. But I am actually not sure if your problem are fixed with it, my changes included a lot of improvements to reduce the amount of shown projects, which shouldn't be there (no budget set, not a project or budget task ...).

I guess using the timeframe (the one in the middle upper screen) won't influence the shown projects? 
If not, that could be your feature request. A switch to change between full project budget (show everything) and only display data that was actually changed in the choosen timeframe (in other words, only show projects that were really worked on).

To sum it up: yes, the budget screen has usability problems. 
W need feeback from users activey using the budget feature, so please add any idea you have on improving it!

**Comment by Skeeve** [comment]
Thanks @kevinpapst 

Seems I don't do anything wrong.

As you assumed: No matter what the filters are, budget shows all. I already created a greasemonkey script to suppress the details of the budgets and just show the summary (I'm not the admin).
